site/docs/spark.md

```diff
@@ -24,7 +24,7 @@ Iceberg uses Apache Spark's DataSourceV2 API for data source and catalog impleme
 | [SQL create table](#create-table)                | ✔️        |            |                                                |
 | [SQL create table as](#create-table-as-select)   | ✔️        |            |                                                |
 | [SQL replace table as](#replace-table-as-select) | ✔️        |            |                                                |
-| [SQL alter table](#alter-table)                  | ✔️        |            |                                                |
+| [SQL alter table](#alter-table)                  | ✔️        |            |  ⚠ Updating partition field or sort order requires extensions enabled  |
```

Comment:
Nit: either "extensions-enabled" or "extensions to be enabled"? Took me some time to parse this sentence correctly...

Reply:
I am following the stored procedure doc that says "requires extensions enabled", maybe "requires extensions enabled to update partition field and sort order" is better